examples/full_example/lib/customs/custom_addon.dart

```diff
@@ -0,0 +1,56 @@
+/// A custom addon example for Widgetbook
+///
+/// [Ref link]: https://docs.widgetbook.io/addons/custom-addon
+import 'package:flutter/material.dart';
+import 'package:widgetbook/widgetbook.dart';
+
+class AlignmentAddon extends WidgetbookAddon<Alignment> {
+  AlignmentAddon({
+    required this.alignment,
+    Alignment? initialAlignment,
+  }) : super(
+          name: 'Alignment Addon',
+          initialSetting: initialAlignment ?? alignment,
+        );
+
+  final Alignment alignment;
```

Comment:
The `alignment` field is redundant, as it does that same thing as the `initialAlignment`

```suggestion
  AlignmentAddon({
    Alignment initialAlignment = Alignment.center,
  }) : super(
          name: 'Alignment',
          initialSetting: initialAlignment,
        );
```